src/app.js

```diff
@@ -507,6 +530,61 @@ const NODE_MAJOR_VERSION = process.versions.node.split('.')[0]
 if (NODE_MAJOR_VERSION < 12) {
 	throw new Error('Requires Node 12 (or higher)')
 }
+// 0, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55
```

Comment:
This section of code is undergoing change - around the way that profiles are activated - currently it has a time check on the hour (the every 5 minutes was for testing). I now think we're going to record an activation time which can then be checked more frequently rather than an active range.
The problem with the active range is how users override which profile is in use.